english/marshmallow.md

```diff
@@ -53,7 +53,7 @@ But you have a growing sense that these mundane objects are vital to your surviv
 
 You decide to try to light a fire to melt some of the marshmallows so you look for some kindling.
 
-You have the suspicion that you are being closely watched.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.